docs/readmes/orc8r/how_to_deploy_using_juju.md

```diff
@@ -0,0 +1,106 @@
+---
+id: how_to_deploy_using_juju
+title: How-To: Deploy Orchestrator using Juju (Beta)
```

Comment:
For consistency with the rest of the docs, I would perhaps remove the "How-To" here, since many other pages fall under this umbrella but don't have this in the name.

Reply:
Apologies for the delay, I was away for a few days.

Sure, that makes sense. Personally, I would prefer to it to be removed for now and for it to be part of a general restructuring, if that's considered desirable by other people too, just so we don't have inconsistencies that are potentially confusing. I don't see it as a blocker though.

Reply:
Understood, I just applied your recommendation.

Reply:
Thanks!